README.md

```diff
@@ -24,332 +39,23 @@ Or install it yourself as:
-# Include the element with id 'selector3' inside the IFrame with id 'frame'
-expect(page).to be_accessible.within '#selector1', '.selector2', iframe: '#frame', selector: '#selector3'
-
-# Nested IFrame selectors
-# Include the element selector1 inside the IFrame with id 'frame2',
-# inside the IFrame with id 'frame1'
-expect(page).to be_accessible.within(iframe: '#frame1', selector: 
-	{iframe: '#frame2', selector: '#selector1'}
-)
-```
-
-The inclusion clause `within '#selector'` specifies which elements of the page should be checked. A valid [CSS selector][css selector] must be provided. Within accepts a single selector, an array of selectors, or a hash describing iframes with selectors.
-
-*see additional [context parameter documentation][context-param]*
-
-#### Exclusion clause
-
-``` ruby
-# Simple selector
-expect(page).to be_accessible.excluding '#selector1'
-
-# Compound selector
-# Exclude all elements with the class 'selector2' inside the element with id 'selector1'
-expect(page).to be_accessible.excluding '#selector1 .selector2'
-
-# Multiple selectors
-# Exclude the element with id 'selector1' *and* all elements with class 'selector2'
-expect(page).to be_accessible.excluding '#selector1', '.selector2'
-
-# IFrame selector
-# Exclude the element with id 'selector1' inside the IFrame with id 'frame1'
-expect(page).to be_accessible.excluding iframe: '#frame1', selector: '#selector1'
-
-# Multiple IFrame selectors
-# Exclude the element with id 'selector1' inside the IFrame with id 'frame1'
-# Exclude the element with id 'selector2' inside the IFrame with id 'frame2'
-expect(page).to be_accessible.excluding(
-	{iframe: '#frame1', selector: '#selector1'},
-	{iframe: '#frame2', selector: '#selector2'}
-)
-
-# Simple selectors with IFrame selector
-# Exclude the element with id 'selector1' *and* all elements with class 'selector2'
-# Exclude the element with id 'selector3' inside the IFrame with id 'frame'
-expect(page).to be_accessible.excluding '#selector1', '.selector2', iframe: '#frame', selector: '#selector3'
-
-# Nested IFrame selectors
-# Exclude the element selector1 inside the IFrame with id 'frame2',
-# inside the IFrame with id 'frame1'
-expect(page).to be_accessible.excluding(iframe: '#frame1', selector: 
-	{iframe: '#frame2', selector: '#selector1'}
-)
-```
-
-The exclusion clause `excluding '#selector'` specifies which elements of the document should be ignored. A valid [CSS selector][css selector] must be provided. Excluding accepts a single selector, an array of selectors, or a hash describing iframes with selectors.
+## Usage
 
-*see additional [context parameter documentation][context-param]*
+### Cucumber
 
-#### Accessibility Standard (Tag) clause
+A set of step definitions have been provided for accessibility testing through Cucumber, using [WebDrivers][webdrivers].
 
-```ruby
-# Single standard
-expect(page).to be_accessible.according_to :wcag2a
+Read the documentation for the [Cucumber API][cucumber-api].
```

Comment:
Instead of Cucumber API, can we say "Cucumber integration" or "usage with Cucumber" or "Cucumber steps" or something?

Feels like a bit of a stretch to say the cucumber steps are an "API".